notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb

```diff
@@ -1236,7 +1259,7 @@
       "source": [
         "## Undeploy the model\n",
         "\n",
-        "When you are done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
+        "When you're done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
```

Comment:
1. There is a typo: "resouce" -> "resource"
2. I think "`Endpoint`" resource should be "endpoint resource"